src/timezones/types.jl

```diff
@@ -222,6 +224,57 @@ function ZonedDateTime(local_dt::DateTime, tz::VariableTimeZone, is_dst::Bool; f
 end
 
 doc"""
+`ZonedDateTime(tz::VariableTimeZone, is_dst::Bool; from_utc::Bool=false) -> ZonedDateTime`
+
+Get a `ZonedDateTime` object in the timezone specified, for the current time.
+
+equivalent to `ZonedDateTime(now(Dates.UTC), tz, is_dst; from_utc=true)`
+"""
+ZonedDateTime(local_dt::DateTime, is_dst::Bool) = ZonedDateTime(now(Dates.UTC), tz, is_dst; from_utc=true)
```

Comment:
You can remove this method. It will be overwritten by: `ZonedDateTime(local_dt::DateTime, is_dst::Bool; from_utc::Bool=false)`. Remember keywords are not included in multiple-dispatch.